cvat/apps/engine/views.py

```diff
@@ -902,6 +903,22 @@ def export_backup(self, request, pk=None):
             )
         return self.serialize(request, backup.export)
 
+    @extend_schema(summary="Aggregate data of a task",
+        responses={
+            '201': OpenApiResponse(description='Consensus Jobs Aggregated'),
+            '202': OpenApiResponse(description='Agreegation of Consensus Jobs started'),
+            '400': OpenApiResponse(description='Agreegating a task without data is not allowed'),
+        },
+    )
+    @action(methods=['PUT'], detail=True, url_path=r'aggregate/?$')
+    def aggregate(self, request, pk=None):
```

Comment:
Let's move this endpoint into the `/api/consensus/` API section. It will be easier to maintain and extend it there in future. Also, please change the method to POST and document the `rq_id` parameter.